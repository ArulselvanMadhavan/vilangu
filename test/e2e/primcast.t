int main() {
  int a;
  Object b;
  b = a;
}