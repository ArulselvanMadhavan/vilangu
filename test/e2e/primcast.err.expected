./test/e2e/primcast.t:4:2 - 4:7: type mismatch. Assignment cast not possible. expecting Object, but got i32
